README.txt

```diff
@@ -0,0 +1,7 @@
+Add custom assets in this directory.
+
+The /css, /fonts, /img, and /js directories are used by USWDS. They have this special pathing to allow easier copy/pasting of components from the USWDS site, and are ignored by git.
+
+Any custom files you need shouuld be in this /assets/custom directory. Like /assets/custom/img/my-cool-image.jpg. This assures that any custom assets aren't ignored by git or overridden by USWDS.
+
+Thanks!
```

Comment:
Any custom assets will not have to go in the `/assets/custom` directory. So I added a note!